app/Http/Controllers/Api/ReportbackController.php

```diff
@@ -68,6 +67,11 @@ public function index(Request $request)
                 $query = $query->where('signups.northstar_id', '=', $filters['northstar_id']);
             }
 
+            // Only return Posts for the given status (if there is one)
+            if (array_has($filters, 'status')) {
+                $query = $query->where('status', '=', $filters['status']);
```

Comment:
As long as we don't have any other APIs relying on the old Phoenix query string that sounds good to me to hold off on it.